elastic/kibana
# [Security Solution][Entity Analytics] Storing input contributions in the risk document

## Summary

This PR adds a new field to the risk document in order to store each risk input contribution score.
The purpose of this is Risk Explainability, particularly in the Risk Contributions expanded panel in the Entity flyouts. 


In addition, we're removing the previous top 5 hits aggregation as we're now obtaining that data through only the scripted metric agg.


### How to test:

1. Ensure you have some entities and alerts. Use the [datagen](https://github.com/elastic/security-documents-generator) tool if needed

2. Turn on the risk Engine in Security -> Mange -> Entity Risk Score

3. Once the task runs check that the risk document via the following request in the Dev Console: `GET risk-score.risk-score-*/_search` 

4. It should now show entities with `input` objects containing a `contribution_score` field.

![Screenshot 2024-03-28 at 14 19 08](https://github.com/elastic/kibana/assets/2423976/c7f649d8-43f3-4ac6-95e9-7fe89d5c03fe)
